src/main/java/org/commcare/suite/model/Detail.java

```diff
@@ -509,6 +515,10 @@ private boolean templatePathValid(String templatePathProvided) {
         return false;
     }
 
+    public boolean doesRequireSearch() {
```

Comment:
I don't love this name, but `isRequireSearch` sounds awkward and `requiresSearch` is confusing.

Reply:
what about `isSearchRequired()`? (swapping the verb and the adjective to match other similar functions)

Reply:
sounds good, 8964eb526